sonar-helm-for-iac/org.sonar.iac.helm/template-evaluation.pb.go

```diff
@@ -0,0 +1,190 @@
+// SonarQube IaC Plugin
```

Comment:
Why this file is included here? I guess it should be generated in "src/org.sonar.iac.helm/" - maybe there is some configuration issue?

Reply:
Or the local `protoc` or `protoc-gen-go` version is not as expected?